### ISSUE TITLE
Vertical scroll is getting stuck

### ISSUE THREAD

**Issue by Achilles718611** [glepur/react-native-swipe-gestures#38]
If I use GestureRecognizer, vertical scroll is getting stuck.
i.e vertical scroll is not flexible.

**Comment by FirdousNath** [comment]
yeah, same! When I use it on top of flatlist , it's not working on iOS. Vertical scroll is no more smooth

**Comment by brkrtp** [comment]
+1

**Comment by hikkary** [comment]
+ 1

**Comment by hikkary** [comment]
+ 1

**Comment by Base29** [comment]
Same here ... any solutions for it ??

**Comment by alopezjs** [comment]
any update here? only happens on iOS
![2](https://user-images.githubusercontent.com/52076780/65786823-1f7b8280-e11d-11e9-96ac-263fa440c454.gif)

**Comment by bartoszboruta** [comment]
@glepur any idea how to fix that?

**Comment by cognitiocorporation** [comment]
Same problem here...

**Comment by ualejandroc** [comment]
The same one here

**Comment by latusdenis** [comment]
@alopezjs Have you found a solution? Facing the same issue

**Comment by inferusvv** [comment]
+1 for iOS. On Android works fine

**Comment by alopezjs** [comment]
Hey everyone, sorry for not answering. I was able to resolve it, don't know if it was a good practice but I just modified the node_modules/react-native-swipe-gestures/index.js file, I commented this section:
```javascript
gestureIsClick(gestureState) {
  return (
    Math.abs(gestureState.dx) < swipeConfig.gestureIsClickThreshold
    // && Math.abs(gestureState.dy) < swipeConfig.gestureIsClickThreshold
   )
}
```

and it works:
![2](https://user-images.githubusercontent.com/52076780/67507780-431ae580-f655-11e9-8d2e-fda3f68fec3d.gif)
still working in android, but I don't know yet if with this I broke anything else, let me know guys if this works for you

**Comment by SSTPIERRE2** [comment]
That fix works for me, but it looks like the change was never released?

**Comment by EatherToo** [comment]
may you  can try my fork to fix this problem  [https://github.com/EatherToo/react-native-swipe-gestures](https://github.com/EatherToo/react-native-swipe-gestures)

**Comment by sajjad-yousaf** [comment]
I have removed this line and the issue is better now, but my screen is still stucking some time while scrolling. 
Please anyone?

**Comment by SlaveWilson** [comment]
@TeamDlxx I think it is because sometimes it detected the event as swipeLeft or swipeRight, I guess increasing the directionalOffsetThreshold should be able to help.
